src/preamble.js

```diff
@@ -1164,6 +1164,16 @@ assert(typeof Int32Array !== 'undefined' && typeof Float64Array !== 'undefined'
 #endif
 
 #if IN_TEST_HARNESS
+
+// Test runs in browsers should always be free from uncaught exceptions. If an uncaught exception is thrown, we fail browser test execution in the REPORT_RESULT() macro to output an error value.
+if (typeof window !== 'undefined') {
+  window.addEventListener('error', function(e) {
+    if (e == 'SimulateInfiniteLoop' || e.message.indexOf('SimulateInfiniteLoop') != -1) return;
```

Comment:
I don't even see that line of code (that checks for SimulateInfiniteLoop in two ways) is even reached in browser.test_sdl_touch..?